pkg/issuer/acme/dns/azuredns/azuredns.go

```diff
@@ -84,7 +114,7 @@ func getAuthorization(env azure.Environment, clientID, clientSecret, subscriptio
 		}
 		return spt, nil
 	}
```

Comment:
It took me a while to realise that the `clientID` parameter will be empty in the case of ambient credentials, but that in that case, the clientID may be supplied via the `managedIdentity` parameter.

Reply:
It's just the same process as for pod managed identities :)